features_module_how_to_share_sites_config_with_oth.md

```diff
@@ -0,0 +1,130 @@
+
+Overridden state means that the configuration in DB is different from configuration in code, and what stored in DB is stronger, so it overrides the code.
+This is a situation we need to solve. We can choose to revert to the original code or to update the Feature by applying the new changes to it. 
+
+
+## Reverting a Feature
+
+Click on the `Overridden` signal, then you will see the component that was override (View in this case because we changed the View title). Check the component you want to revert and click on `Revert components`
+
+![](images/features/revert.png)
+
+
+Now go back to Features administration page and you will see that the state is `Default`, means the feature configurations in code are the same like in DB.
+
+![](images/features/default_state.png)
+
+
+## Updating a Feature
+
+When we make changes to the configuration and want to keep them, we need to update the code.
+One way to do this is through the UI: click on `recreate` and download the Feature again. Then replace the old Feature folder in module directory with the new one.
+A second and much quicker way is to use Drush. Run `drush fu [feature name]` in Drupal directory (www) and that's it, your Feature is updated!
+
+If you go to Features administration page, you will see that the state is `Default` again.
+
+
+## What to “Featurize”? 
+
+In short: everything that is _not_ content!
+
+Everything we have in our local DB should be in code, so we can share it.
+
+Of course we can't Featurize content. Content needs to be [migrated](https://www.drupal.org/project/migrate).
+
+How do we decide which components to pack in a feature? Feature is a set of components for a particular use case, so we need to create Features using common sense.
+
+For example, if we have a site with a main menu that links to: 
+1. Gallery - made of content type, Views and Vocabulary (taxonomy)
+2. Blog - made of content type, Views and Vocabulary (taxonomy)
+3. About page - made of content type.
+
+How many Features are we going to create from this structure?
+
+We can pack everything into one module, but it can be difficult to maintain.  If one person works on the Gallery, and the other makes changes to the Blog, both of them need to recreate the same Feature. So, it makes more sense to pack the Gallery into one Feature and the Blog in another.
+
+![](images/features/two_features.png)
+
+But, what if both Gallery and Blog use the same Vocabulary? If we pack the Vocabulary with the Gallery Feature, and also with the Blog Feature, we will get a conflict when we try to enable them both. So, it looks like the right strategy in this case is to pack the Vocabulary itself into a separate Feature and define the Vocabulary Feature as a dependency in the Gallery Feature and in the Blog Feature.
+
+![](images/features/three_features.png)
+
+What about the Menu? Well, we can also pack it as a separate Feature, or we can treat the menu as content and not Featurize it at all.
```

Comment:
In D8 menu it content, so I'd recommend here to migrate it, but mention that in D7 we could bundle it in a feature as-well.